google/cloud/sql/connector/connector.py

```diff
@@ -65,5 +67,6 @@ def connect(instance_connection_string, driver: str, **kwargs):
     # Return a DBAPI connection
 
     loop = _get_loop()
-    icm = InstanceConnectionManager(instance_connection_string, loop)
+    user_agent = f"cloud-sql-python-connector-{driver}/{__version__}"
+    icm = InstanceConnectionManager(instance_connection_string, user_agent, loop)
```

Comment:
Do we need to pass version in as an argument, or can we just import it directly?

Reply:
you mean import it directly in the instanceConnectionManager file?

Reply:
Refactored this so that we just pass in the driver.